manager/container.go

```diff
@@ -270,6 +271,30 @@ func (c *containerData) updateLoad(newLoad uint64) {
 	}
 }
 
+func (c *containerData) updateSubcontainerStats(stats *info.ContainerStats, path string) (*info.ContainerStats, error) {
+	// type []info.ContainerReference
+	subcontainers, err := c.handler.ListContainers(container.ListRecursive)
+	// glog.Infof("subcontainers for container %v are %v", c.info.Name, subcontainers)
+	if err != nil {
+		// glog.Infof("returning from subcontainerstats with error %v", err)
+		return stats, err
+	}
+	for _, sub := range subcontainers {
```

Comment:
Another problem with this approach is that some of the subcontainers might be idle and we would end up checking load for them. Today,  manager's housekeeping slows down stats check for idle containers.

This is more of a concern on systemd systems, where there are billion containers created at startup.